.github/workflows/shell-app-android.yml

```diff
@@ -1,11 +1,7 @@
 name: Android Shell App
 
 on:
-  workflow_dispatch:
-    inputs:
-      releaseShellAndroid:
-        description: 'type "release-shell-android" to confirm upload'
-        required: false
+  workflow_dispatch: {}
```

Comment:
is workflow_dispatch necessary here?

Reply:
If it's not here the button in Github Actions UI that dispatches the workflow isn't shown.

Reply:
... and it wouldn't be possible to trigger it through GitHub API, so `et workflow-dispatch` wouldn't work for this workflow

Reply:
Interestingly I think I have been successfully triggering workflows that don't have any `workflow_dispatch` via `et`. Not saying your statement isn't true, just noting this may be possible.